docs/usage/configuration-options.md

```diff
@@ -754,7 +754,7 @@ Example:
 
 Ignore the default project level approval(s), so that Renovate bot can automerge its merge requests, without needing approval(s).
 Under the hood, it creates a MR-level approval rule where `approvals_required` is set to `0`.
-This option works only when `automerge=true`, `automergeType=pr` and `platformAutomerge=true`.
+This option works only when `automerge=true`, `automergeType=pr or branch` and `platformAutomerge=true`.
```

Comment:
```suggestion
This option works only when `automerge=true`, `automergeType=pr` or `automergeType=branch` and `platformAutomerge=true`.
```

It might be clearer to list the precise config options, instead of having them both in one go: `automergeType=pr or branch`.